Fika.Core/Coop/Components/FikaServerStunQuery.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections;
+using UnityEngine;
+using Fika.Core.Networking.NatPunch;
+using Comfort.Common;
+using Fika.Core.Networking;
+
+namespace Fika.Core.Coop.Components
+{
+    public class FikaServerStunQuery : MonoBehaviour
+    {
+        private Coroutine stunQueryRoutine;
+
+        public void StartServerStunQueryRoutine()
+        {
+            stunQueryRoutine = StartCoroutine(StunQuery());
+        }
+
+        public void StopServerStunQueryRoutine()
+        {
+            if (stunQueryRoutine != null)
+            {
+                StopCoroutine(stunQueryRoutine);
+                stunQueryRoutine = null;
+            }
+        }
+
+        private IEnumerator StunQuery()
+        {
+            while (true)
+            {
+                var fikaServer = Singleton<FikaServer>.Instance;
```

Comment:
```suggestion
                FikaServer fikaServer = Singleton<FikaServer>.Instance;
```
Explicit variables so that we stick to the standard of the repo.